hack/verify-generated-files.sh

```diff
@@ -18,12 +18,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export KUBE_ROOT
```

Comment:
Not that you changed this, but I wonder why this is necessary?

Reply:
if you assign and `export` on the same line, the `export` will swallow the exit code from `$(...)`, which defeats `set -o errexit`. same thing for `local foo=$(bar ...)`

Reply:
splitting this onto two lines ensure that if `dirname` errors we don't ignore it